File: 프로그래머스/코딩테스트 입문/영어/k의 개수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int i, int j, int k) {
    int answer = 0;
    string kStr = to_string(k);

    for (int num = i; num <= j; num++) {
        string numStr = to_string(num);

        for (char c : numStr) {
            if (c == kStr[0]) {
                answer++;
            }
        }
    }

    return answer;
}

File: 프로그래머스/코딩테스트 입문/영어/k의 개수.java
class Solution {
    public int solution(int i, int j, int k) {
        int answer = 0;
        String kStr = Integer.toString(k);

        for (int num = i; num <= j; num++) {
            String numStr = Integer.toString(num);

            for (char c : numStr.toCharArray()) {
                if (c == kStr.charAt(0)) {
                    answer++;
                }
            }
        }

        return answer;
    }
}